spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -351,6 +351,8 @@ abstract class Graph[T: ClassTag](
       })
 
     if (curNode.element.output.isTable) {
+      output.toTable
```

Comment:
Why do you change this?

Reply:
Forgot to undo changes in Graph. There should be no changes in Graph now.